### PR TITLE
Issue #2375. Investigate intermittent Travis failures.

### PR DESCRIPTION
**Pull request by miketaylr**
I'm gonna use this PR to try to debug why Travis fails to start at times.

Please ignore.

### CODE REVIEW

**Comment by miketaylr** [comment]
Upgraded Travis because it contains a bugfix for https://github.com/theintern/intern/issues/868.

**Comment by miketaylr** [comment]
Lets go ahead and land this? I'd rather not re-start the build 20 times to find a failure. This way we can try to catch one in the wild.

r? @karlcow

**Comment by miketaylr** [comment]
(not not using "Fixes" in title, because this doesn't fix it...)

File: tests/functional/_intern.js [hunk 1]
```diff
@@ -96,8 +104,19 @@ if (args.showBrowser) {
 
 intern.configure(config);
 
-intern.run().catch(e => {
-  // This might not throw, BUG filed: https://github.com/theintern/intern/issues/868
+try {
+  intern.run().then(
+    () => {
```
Comment (karlcow) [inline tests/functional/_intern.js]: What does that mean? Just for my education. :)

**Pull request merged**